tree-sitter-darklang/grammar.js

```diff
@@ -70,9 +70,26 @@ module.exports = grammar({
       seq(
         field("keyword_type", alias("type", $.keyword)),
         field("name", $.type_identifier),
+        optional(
+          seq(
+            field("symbol_open_angle", alias("<", $.symbol)),
+            field("type_params", $.type_decl_type_params),
+            field("symbol_close_angle", alias(">", $.symbol)),
+          ),
+        ),
```

Comment:
the syntax to apply type args will be the same for many places (`DB<Int>`, `myFn<Int>`, `MyType<Int>`) - let's extract this out to a `type_args` root-level thing, and share across the use cases.